src/modules/auth/index.js

```diff
@@ -188,6 +179,36 @@ export default class AuthModule extends Module {
 		});
 	}
 
+
+	async validateNavigation() {
+		let getItems = this.Navigation.getItems();
```

Comment:
`this.Navigation` should be replaced by `this.app.Navigation` ... it respects the ownership.

Reply:
@ateska I suppose it does, its a global value set in the constructor: https://github.com/TeskaLabs/asab-webui/blob/fad6b15f8e26db2a4017f831dbcac5562cf0da59/src/modules/auth/index.js#L25

Anyway, if it should be changed by your proposal, then we have to refactor it elswhere, e.g.:
https://github.com/TeskaLabs/asab-webui/blob/fad6b15f8e26db2a4017f831dbcac5562cf0da59/src/modules/auth/index.js#L19-L21

or not?

Reply:
Well, yes ... it is a better practice to use `this.App.XYZ` instead of `this.XYX = this.App.XYZ` because people knows the app structure from elsewhere.